src/NzbDrone.Core/Download/Clients/DownloadStation/TorrentDownloadStation.cs

```diff
@@ -272,7 +272,14 @@ protected ValidationFailure TestOutputPath()
         {
             try
             {
-                var downloadDir = GetDownloadDirectory();
+                var downloadDir = GetDefaultDir();
+
+                if (downloadDir == null)
+                {
+                    return new NzbDroneValidationFailure(string.Empty, "No default destination configured. You must manually set it up into download station settings.");
```

Comment:
string.Empty => 'TvDirectory' that way the message appears next to the TvDirectory.
Also keep the message as short as possible (that way it fits in the whitespace), move a more advanced explanation to the DetailedDescription property.